tests/queries/0_stateless/02884_string_distance_function.sql

```diff
@@ -51,5 +51,7 @@ SELECT stringJaccardIndexUTF8(materialize('hello'), materialize('\xC0\x80')); --
 SELECT stringJaccardIndexUTF8(materialize('hello'), materialize('\xD8\x00 ')); -- { serverError BAD_ARGUMENTS }
 SELECT stringJaccardIndexUTF8(materialize('hello'), materialize('\xDC\x00')); -- { serverError BAD_ARGUMENTS }
 SELECT stringJaccardIndexUTF8('😃🌍', '🙃😃🌑'), stringJaccardIndex('😃🌍', '🙃😃🌑');
+SELECT editDistanceUTF8('我是谁', 'Tom');
+SELECT editDistanceUTF8('我是谁', '我是我');
```

Comment:
How about add a test in the '-- non-const arguments' part above, just after the line 33?
Like the function stringJaccardIndexUTF8 did.

Reply:
ok